gqcpy/src/QCMethod/HF/UHF/UHFSCFEnvironment_bindings.cpp

```diff
@@ -103,33 +103,27 @@ void bindQCMethodUHFSCFEnvironmentInterface(Class& py_class) {
          */
 
         .def("replace_current_coefficient_matrices",
```

Comment:
Maybe you could also rename this to `replace_current_coefficient_matrix`?

Reply:
Allright! I edited all UHF API's to reflect the other Hartree-Fock methods.